src/components/Forms/MaskedTextField.js

```diff
@@ -275,7 +275,12 @@ class MaskedTextField extends React.Component {
             snacksTheme={snacksTheme}
           />
 
```

Comment:
Is `MaskedTextField` the only text field that needs this? There's another `TextField` component, not sure if that's relevant here.

Reply:
Good call, didn't see TextField using hint. Will update if needed